cardano-cli/test/cardano-cli-golden/Test/Golden/Latest/Transaction/CalculateMinFee.hs

```diff
@@ -18,10 +22,37 @@ import Hedgehog.Extras qualified as H
 
 {- HLINT ignore "Use camelCase" -}
 
+hprop_golden_latest_transaction_calculate_min_fee :: Property
+hprop_golden_latest_transaction_calculate_min_fee =
+  watchdogProp . propertyOnce $ do
+    protocolParamsJsonFile <-
+      noteInputFile
+        "test/cardano-cli-test/files/input/protocol-params-preview.json"
```

Comment:
You are cross-referencing from golden test suite a file in `test` test suite. This fails in nix: https://ci.iog.io/build/8132079/nixlog/1 and annoying to work around nix files (you need to add each input file manually there) and maintain. It's just easier to have two identical files in the repo.